platform/security/expansion/security-expansion-commands/src/main/java/ddf/security/command/ExpansionsCommand.java

```diff
@@ -41,10 +41,8 @@ protected Object doExecute() throws Exception {
                 Map<String, List<String[]>> map = expansion.getExpansionMap();
                 System.out.print(Ansi.ansi().fg(Ansi.Color.YELLOW).toString());
                 if ((map != null) && (!map.isEmpty())) {
-                    for (String key : map.keySet()) {
-                        for (String[] mapping : map.get(key)) {
-                            System.out.println(key + " : " + mapping[0] + " : " + mapping[1]);
-                        }
+                    for(Map.Entry<String, List<String[]>> entry : map.entrySet()) {
```

Comment:
This logic is different now. It will only get the first tuple out of the `List<String[]>`. If there are multiples in there, they'll be ignored.

The same logic would be:
```java
for(Map.Entry<String, List<String[]>> entry : map.entrySet()) {
    for (String[] mapping : entry.getValue()) {
        System.out.printf("%s : %s : %s%n", entry.getKey(), mapping[0], mapping[1]);
    }
}
```